src/app/components/login/login.component.ts

```diff
@@ -29,10 +32,11 @@ export class LoginComponent implements OnInit {
     };
 
     this.loginService.postLoginAttempt(userLogin).subscribe(
-      (user: User) => console.log('I GOT THIS USER!!!', user),
+      (user: User) => sessionStorage.setItem("user",JSON.stringify(user)),
```

Comment:
Change this line so that it uses the session keys in our Constants class (referenced in my comment above).

It'll end up looking like 
`sessionStorage.setItem(Constants.sessionKeys.user,JSON.stringify(user)),`